src/core/cache/tables.sql

```diff
@@ -0,0 +1,44 @@
+-- Tables schema definition for watchit cache storage.
+-- ref: https://www.sqlstyle.guide/
+-- Schema table definition for movies.
+CREATE TABLE IF NOT EXISTS movies (
+    movie_id INTEGER PRIMARY KEY,
+    -- imdb code is adopted from IMB movies site to handle an alpha-numeric id
+    imdb_code TEXT KEY DESC,
```

Comment:
if these identifiers are fixed length you may consider VARCHAR

Reply:
sqlite doesnt support it :( 
https://sqlite.org/matrix/datatype3.html